yt_dlp/extractor/youtube/pot/builtin/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .cache.memory import MemoryLRUPCP as _MemoryPCP  # noqa: F401
```

Comment:
Do we need this whole package hierarchy?

`pot.utils`, `pot.builtin_cache` are sufficient, no?

Reply:
I think I'll keep the builtin hierarchy but flatten it.

We will probably be adding PO token providers in there later on